### PR TITLE
fix build on archlinux (pkg-config cblas)

### PR DESCRIPTION
**Pull request by vbmithr**

### CODE REVIEW

File: src/owl/config/configure.ml [hunk 1]
```diff
@@ -182,6 +182,11 @@ the output of `src/owl/config/configure.exe --verbose`.
 
 let () =
   C.main ~name:"owl" (fun c ->
+      let cblas_conf =
+        let default = { C.Pkg_config.cflags = [] ; libs = ["-cblas"] } in
```
Comment (mseri) [inline src/owl/config/configure.ml]: I think there you meant `-lcblas` but in this case I would leave that empty, as in most cases we want to use the `cblas` that comes from `openblas` instead (and `-lcblas` used to break some of the builds when I was refactoring the build process and trying to fix ubuntu).

Comment (mseri) [reply src/owl/config/configure.ml]: Also, I think you should add a test source with `#include <cblas.h> ...` and use that to detect if the `cblas` flags are actually needed.

**Comment by ryanrhymes** [comment]
@mseri are you happy with the revision, do you think we can merge this now?

**Comment by mseri** [comment]
I think it is fine to start with it, we can add more fine checks on when cblas linking is actually needed in a separate PR if it ever becomes necessary

**Pull request merged**